src/commands/slack-notify-waiting-for-approval.yml

```diff
@@ -57,16 +57,16 @@ steps:
 - run:
     name: Set variables
     command: |
-      echo "export EMAIL_DOMAIN=<<parameters.email_domain>>" >> "$BASH_ENV"
-      echo "export SLACK_BOT_TOKEN=<<parameters.slack_bot_token>>" >> "$BASH_ENV"
-      echo "export SLACK_MESSAGE=<<parameters.slack_message>>" >> "$BASH_ENV"
+      echo 'export EMAIL_DOMAIN="<<parameters.email_domain>>"' >> "$BASH_ENV"
```

Comment:
I'm guessing string interpolation happens before the command is run, so I _think_ it will still properly interpolate the parameter even though it's in single quotes, but I don't know how to test this

Reply:
I think this should work -- an alternate would be to escape the inner quotes:
```bash
echo "export EMAIL_DOMAIN=\"<<parameters.email_domain>>\"" >> "$BASH_ENV"
```